google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -2409,13 +2409,13 @@ public void testBucketPolicyV1RequesterPays() throws ExecutionException, Interru
     storage.create(BucketInfo.newBuilder(bucketName).build());
 
     try {
-      Bucket remoteBucket =
+      Bucket bucketDefault =
           storage.get(
               bucketName, Storage.BucketGetOption.fields(BucketField.ID, BucketField.BILLING));
-      assertNull(remoteBucket.requesterPays());
-      remoteBucket = remoteBucket.toBuilder().setRequesterPays(true).build();
-      Bucket updatedBucket = storage.update(remoteBucket);
-      assertTrue(updatedBucket.requesterPays());
+      assertNull(bucketDefault.requesterPays());
+
+      Bucket bucketTrue = storage.update(bucketDefault.toBuilder().setRequesterPays(true).build());
```

Comment:
requesterPaysBucket? modifiedBucket?

Reply:
To me bucketDefault, bucketTrue, bucketFalse sound more informative than bucket, modifiedBucket, modifiedBucket2. But I'm okay with any names.